docs/api/locking.md

```diff
@@ -0,0 +1,449 @@
+# Git LFS File Locking API
+
+Added: v2.0
+
+The File Locking API is used to create, list, and delete locks, as well as
+verify that locks are respected in Git pushes. The locking URLs are built
+by adding a suffix to the LFS Server URL.
+
+Git remote: https://git-server.com/foo/bar  
```

Comment:
Markdown tables are so hard to read and write in raw text, I prefer to use them only when necessary. A two column table isn't necessary.